lighthouse-cli/test/check-formatting.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+diff -u <(cat *.ts) <(./node_modules/.bin/clang-format -style=file *.ts)
```

Comment:
wanna redirect output to devnull so we dont see anything?
![image](https://cloud.githubusercontent.com/assets/39191/25791540/032cee4a-3376-11e7-80d6-473faf8297cf.png)